src/west/app/project.py

```diff
@@ -863,11 +868,9 @@ def update(self, project):
             # out the new detached HEAD, then print some helpful context.
             if take_stats:
                 start = perf_counter()
-            project.git('checkout --detach --quiet ' + sha)
+            project.git('checkout --detach ' + sha)
```

Comment:
unsure if this could become too noisy to some users.
But, only one way to find out.

Alternative could be to do this when `-v` is given.

Reply:
It's the same number of lines of output when the tree is clean